zksync_sdk/wallet.py

```diff
@@ -227,20 +262,23 @@ async def build_withdraw_nft(self,
     def address(self):
         return self.eth_signer.address()
 
-    async def build_transfer(self, to: str, amount: Decimal, token: TokenLike,
-                             fee: Decimal = None,
-                             valid_from=DEFAULT_VALID_FROM,
-                             valid_until=DEFAULT_VALID_UNTIL) -> Tuple[Transfer, TxEthSignature]:
-        account_id, nonce = await self.zk_provider.get_account_nonce(self.address())
-        token = await self.resolve_token(token)
-        if fee is None:
-            fee = await self.zk_provider.get_transaction_fee(FeeTxType.transfer, to, token.id)
-            fee = fee.total_fee
-        else:
-            fee = token.from_decimal(fee)
+    async def build_transfer(
+        self, 
+        to: str, 
+        amount_wei: int, 
```

Comment:
Usually, in the `transfer` function and the other the amount that is passed is in `Decimal` (so basically 1 USDC there would be `1`, but here we need wei, e.g. the `1000000`. The difference should be clear because of the different name

Reply:
Why we can't just receive Decimal?

Reply:
The difference is not clear.